tests/Doctrine/Tests/PHPUnitCompatibility/OnNotSuccessfulTestCompat.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\PHPUnitCompatibility;
+
+use Throwable;
+
+use const PHP_VERSION_ID;
+
+if (PHP_VERSION_ID >= 80100) {
+    trait OnNotSuccessfulTestCompat
+    {
+        protected function onNotSuccessfulTest(Throwable $e): never
```

Comment:
This change of return type is a PITA. 😅

Reply:
Good thing we're dropping ORM 2 soon :slightly_smiling_face:

Reply:
I sure hope that day will come.